app/styles/components/_imaging.scss

```diff
@@ -0,0 +1,2 @@
+.imaging-row .notes { max-width: 300px; }
+.completed-imaging-row .notes { max-width: 300px; }
```

Comment:
@cacqw7 not sure why we need two different classes here.  @jglovier any opinion on using two different classes for completed vs in-progress?